src/app/core/components/header/user-panel/user-panel.component.ts

```diff
@@ -8,6 +14,17 @@ import { faSignOutAlt, faUser, IconDefinition } from '@fortawesome/free-solid-sv
   styleUrls: [ './user-panel.component.scss' ],
 })
 export class UserPanelComponent {
+  @Input() userName$: Observable<string>;
```

Comment:
no need to use this Observable input. Just use the service.

Reply:
Okay